### PR TITLE
Wolvesnotbear

### PR DESCRIPTION
**Pull request by psykzz**
Renames Iron Bears to Steel Wolves

### CODE REVIEW

File: code/modules/clothing/under/marine_uniform.dm [hunk 1]
```diff
@@ -178,8 +178,8 @@
 	has_sensor = 0
 
 /obj/item/clothing/under/marine/veteran/bear
```
Comment (DominikPanic) [inline code/modules/clothing/under/marine_uniform.dm]: Change the typepaths too

File: code/modules/clothing/suits/marine_armor.dm [hunk 2]
```diff
@@ -640,8 +640,8 @@
 //===========================//DISTRESS\\================================
 
 /obj/item/clothing/suit/storage/marine/veteran/bear
-	name = "\improper H1 Iron Bears vest"
-	desc = "A protective vest worn by Iron Bears mercenaries."
+	name = "\improper H1 Steel Wolvess vest"
```
Comment (DominikPanic) [inline code/modules/clothing/suits/marine_armor.dm]: There's 2 s everywhere

File: _maps/shuttles/tgs_canterbury.dmm [hunk 3]
```diff
@@ -35,7 +35,7 @@
 /area/shuttle/canterbury/cic)
 "ag" = (
 /obj/structure/table/mainship,
-/obj/machinery/computer/station_alert,
+/obj/machinery/faxmachine/cic,
```
Comment (DominikPanic) [inline _maps/shuttles/tgs_canterbury.dmm]: Unrelated change

**Comment by DominikPanic** [comment]
Get the lore master to approve it

**Comment by grinkgo** [comment]
ok but why

File: code/modules/clothing/suits/marine_armor.dm [hunk 4]
```diff
@@ -639,9 +639,9 @@
 
 //===========================//DISTRESS\\================================
 
-/obj/item/clothing/suit/storage/marine/veteran/bear
-	name = "\improper H1 Iron Bears vest"
-	desc = "A protective vest worn by Iron Bears mercenaries."
+/obj/item/clothing/suit/storage/marine/veteran/wolves
+	name = "\improper H1 Steel Wolves vest"
+	desc = "A protective vest worn by Steel Wolves mercenaries."
 	icon_state = "bear_armor"
```
Comment (DominikPanic) [inline code/modules/clothing/suits/marine_armor.dm]: If you really wanted you could also fix the icon states, though not required

Comment (psykzz) [reply code/modules/clothing/suits/marine_armor.dm]: I'll fix it tonight, its a fair ask, and id ask someone else to do it too.

**Comment by Rohesie** [comment]
I assume the intent is to reduce Aliens-based lore and replace it with our own: https://avp.fandom.com/wiki/Iron_Bears
This is the kind of thing I'd ask input to the Russian community and see if they wouldn't want to be represented in a different way. If they have non-meme suggestions that would be nice.

**Comment by psykzz** [comment]
@Rohesie I don't know who in the community to ask, so hard for me to know. If you have someone you want to reach out to, happy to wait.

**Comment by Pariah919** [comment]
I'm pretty sure I was told the Iron Bears were copyrighted.
Which is why I rewrote their name and general lore.
The idea was about this- A PMC vaguely inspired by cold war Soviet gear and attitude to most things.
They're not strictly Russian. Just mostly I guess.
